euclidean_gcd/CPP/euclidean_gcd.cpp

```diff
@@ -0,0 +1,24 @@
+#include <bits/stdc++.h>
+#define ll long long
+using namespace std;
+
+ll gcd(ll a, ll b)
+{
+    if (a == 0)
+        return b;
+    if (b == 0)
+        return a;
+    return gcd(b % a, a);
+}
+
+int main()
+{
```

Comment:
{ should almost always be at the end of the previous line  [whitespace/braces] [4]

*Origin: CPPLintBear, Section: `all.cpp`.*